jbpm-workitems/jbpm-workitems-rest/src/test/java/org/jbpm/process/workitem/rest/RestWorkItemHandlerTest.java

```diff
@@ -914,6 +934,44 @@ public void testPUTOperationWithDefaultCharset() {
                      responseMsg);
     }
 
+    @Test
+    public void testPUTOperationWithSetCharset() {
```

Comment:
This test looks like a duplicate of `testPUTOperationWithCharsetSpecified`. I think we can leave `testPUTOperationWithCharsetSpecified` (just rename it to CharsetUnspecified) as it was and this newly added test can test the new way.

Reply:
updated - thanks!